pycode/memilio-epidata/memilio/epidata_test/test_epidata_get_vaccination_data.py

```diff
@@ -293,14 +293,15 @@ def test_split_column_based_on_values_ident_length_wrong(self):
                     column_names_i[j],
                     'Column names do not match.')
 
-    @unittest.skip("Skip Test till create_intervals_mapping is fixed")
-    def test_intervall_mapping(self):
-        lower_bounds1 = np.array(
+    def test_interval_mapping(self):
```

Comment:
Maybe add some tests for edge cases where the bounds of the intervalls don't match

Reply:
Added a test case with invalid boundaries and changed one to have non matching boundaries.